flaky test in the flashback_test

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->

### 2. What did you expect to see? (Required)

### 3. What did you see instead (Required)

```
panic: region 40 is in flashback progress, FlashbackStartTS is 449658828228919302



goroutine 175816 [running]:

github.com/pingcap/tidb/pkg/infoschema.(*infoschemaV2).SchemaTables(0xc018854c00, {{0xc01fb590fc?, 0xc007625680?}, {0xc01fb5910c?, 0xc021880180?}})

	pkg/infoschema/infoschema_v2.go:600 +0x512

github.com/pingcap/tidb/pkg/ddl.(*ddl).refreshTiFlashTicker(0xc0207c50e0, {0xa070590, 0xc007625680}, 0xc021880180)

	pkg/ddl/ddl_tiflash_api.go:455 +0x623

github.com/pingcap/tidb/pkg/ddl.(*ddl).PollTiFlashRoutine(0xc0207c50e0)

	pkg/ddl/ddl_tiflash_api.go:588 +0x9c5

github.com/pingcap/tidb/pkg/util.(*WaitGroupWrapper).Run.func1()

	pkg/util/wait_group_wrapper.go:157 +0x9e

created by github.com/pingcap/tidb/pkg/util.(*WaitGroupWrapper).Run in goroutine 175139

	pkg/util/wait_group_wrapper.go:155 +0xf1

```
### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->

--- a/pkg/infoschema/infoschema_v2.go
+++ b/pkg/infoschema/infoschema_v2.go
@@ -17,7 +17,9 @@ package infoschema
 import (
 	"fmt"
 	"math"
+	"strings"
 	"sync"
+	"time"
 
 	"github.com/pingcap/errors"
 	"github.com/pingcap/tidb/pkg/ddl/placement"
@@ -582,6 +584,7 @@ func (is *infoschemaV2) SchemaTables(schema model.CIStr) (tables []table.Table) 
 		return tables
 	}
 
+retry:
 	dbInfo, ok := is.SchemaByName(schema)
 	if !ok {
 		return
@@ -595,6 +598,13 @@ func (is *infoschemaV2) SchemaTables(schema model.CIStr) (tables []table.Table) 
 	if err != nil {
 		if meta.ErrDBNotExists.Equal(err) {
 			return nil
+		}
+		// Flashback statement could cause such kind of error.
+		// In theory that error should be handled in the lower layer, like client-go.
+		// But it's not done, so we retry here.
+		if strings.Contains(err.Error(), "in flashback progress") {
+			time.Sleep(200 * time.Millisecond)
+			goto retry
 		}
 		// TODO: error could happen, so do not panic!
 		panic(err)

